src/apic-extension/azext_apic_extension/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
-    "azext.isPreview": true,
+    "azext.isPreview": false,
```

Comment:
from https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md, i found:
> For all stable extension releases, neither azext.isPreview nor azext.isExperimental can be added into module metadata.

Reply:
Thank you, removed the property.